migra/formatter.py

```diff
@@ -0,0 +1,17 @@
+import re
+
+class Formatter:
+
+    def __init__(self, eager_parens: bool=True) -> None:
+        # add other format wide configs
+        self.eager_parens = eager_parens
+
+
+    def create_policy(self, all_changes):
+        if self.eager_parens:
```

Comment:
tf is the point of `eager_parens`

document the attribute or toss it, see no point

Reply:
Debugging mainly. I was concerned about the robustness of the regex and wanted to be able to easily turn this flag on or off. But I can see why it's unnecessary. I had an understanding that this was more of a bug fix and so thought some of the sloppy elements were ok for the sake of debugging flow ease of use.